### DISCUSSION TITLE
Connection problem with cassandra on python

### DISCUSSION THREAD

**Discussion by wass0809** [community/community#83613] [Programming Help]

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @wass0809 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
